docs/developer-guide/ksqldb-clients/python-client.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: page
+title: Python Client for ksqlDB
+tagline: Python client for ksqlDB
+description: Send requests to ksqlDB from your Python app
+keywords: ksqlDB, python, client
+---
+
+The Python client enables sending requests easily to a ksqlDB server
+from within your Python application, as an alternative to using the [REST API](../api.md).
+The client only supports pull and push queries.
+
+!!! tip
+    [View the Python client API documentation](api/BLAHBLAHNEEDTOCHANGE.html)
```

Comment:
@JimGalasyn How can I link this to [docs/python-client/html/ksqldb_confluent/index.html](https://github.com/confluentinc/ksql/pull/9265/files#diff-d6d8b766aa07f72826529755dca2cb1a5965ec92e81c029240b0af5a948c093a)?